File: vectors/Code challenge 1.py
#Code challenge
import numpy as np
m=4
n=6


a = np.random.randn(m,n)
b = np.random.randn(m,n)
dp = np.zeros(n)

for i in range(n):
		dp[i] = np.dot(a[0:,i],b[0:,i])

print (dp)

File: vectors/Code_challenge_1.m
# Code challenge
a = randn(4,6);
b = randn(4,6);

l = size(a,2); %This computes to 6
dp = zeros(l,1); %a row vector with 6 zeros

for i = 1:l  %going from 1 to 6
    dp(i) = dot( a(:,i),b(:,i) );  %dot product of i'st column of a and i'st column of b
end

disp ([dp])

